semantic-conventions/CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ Please update the changelog as part of any significant pull request.
 
 ## Unreleased
 
-- BREAKING: Introduced attribute requirement levels ([#92](https://github.com/open-telemetry/build-tools/pull/92)):
+- Add changes here.
+
+## v0.12.0
```

Comment:
@open-telemetry/technical-committee @open-telemetry/build-tools-approvers
Do you think we should wait for #79 to be merged and release both changes at once or release v0.12 now and v0.13 with metrics support soon-ish?

Reply:
As discussed on the OTel spec call: No objections to releasing v0.12.0 right away as-is.